src/UXClient/Components/GeoProcessGraphic/GeoProcessGraphic.scss

```diff
@@ -0,0 +1,106 @@
+@import "../../styles";
+
+@mixin tColors($grays){
+    $gray1: nth($grays, 1); // text
+    $gray2: nth($grays, 2); // hover
+    $gray3: nth($grays, 3); // element
+    $gray4: nth($grays, 4); // background
+    $gray5: nth($grays, 5); 
+    $gray6: nth($grays, 6); // hover
+
+    box-shadow: 0 1px 3px 0 rgba($gray1, .16);
+    border-color: rgba($gray1, .1);
+    
+    @include chartColors($grays);
+
+    color: $gray1;
+    background-color: $gray3;
+
+    .tsi-gpgValueTable tr {
+        border-bottom: 1px solid rgba($gray1, .1);
+        &:last-of-type {
+            border-bottom: none;
+        }
+        border-left-width: 10px;
+        border-left-style: solid;
+    }
+
+}
+.tsi-gpgTooltipTitle{
+    @extend .tsi-textWrap;
+}
+.tsi-gpgTooltipTitle {
+    font-weight: bold;
+    font-size: 13px;
+} 
+.popup-container{
```

Comment:
prefix with tsi- unless this is an az maps style

Reply:
it's an azure maps style